### DISCUSSION TITLE
Option to allow all bookmark symbols to stay the same

### DISCUSSION THREAD

**Discussion by amo-f77** [organicmaps/organicmaps#740] [Ideas]
I would like the option to show all my bookmarks/pins with the same standard symbol with the star inside. Like the one shown when I choose the colour.
Currently the bookmarks are shown with a mix of the original star and symbols for the type of poi. They get the tag of icon, which then shows the relevant icon/poi symbol, instead of the bookmark star.

See below for the an example. I have ticked and crossed to show what I mean. If they were all the star type, it would be easier to see them when zoomed out a bit more. They wouldn't get lost with other pois.
![om](https://user-images.githubusercontent.com/76026031/123504558-6ead6900-d69d-11eb-80b5-c595b6c247a1.jpg)

**Comment by Shohreh** [comment]
It'd be even better to be able to…
1. Apply/change the color for a whole set of bookmarks in one go, somehing Maps.me never did
2. Choose a symbol for a set of bookmarks, in addition to a color

**Reply by csidirop** [reply]
I'd like the idea of (optionally) setting one main color for a set of bookmarks.
